api-put-object.go

```diff
@@ -147,8 +148,12 @@ func (c Client) putObjectCommon(ctx context.Context, bucketName, objectName stri
 		return c.putObjectNoChecksum(ctx, bucketName, objectName, reader, size, opts)
 	}
 
+	if opts.PartSize == 0 {
+		opts.PartSize = minPartSize
```

Comment:
Should `minPartSize` be renamed as `defaultPartSize`? otherwise LGTM. Ttested the PR with various part sizes.

Reply:
Not in this PR.